data_steward/test/unit_test/validation_test.py

```diff
@@ -93,6 +95,27 @@ def test_all_files_unparseable_output(self, mock_check_cron):
                 expected = f.read()
                 self.assertEqual(expected, actual_result)
 
+    @mock.patch('api_util.check_cron')
+    def test_bad_file_names(self, mock_check_cron):
+        exclude_file_dic_list = resources._csv_to_list(WARNING_UNKNOWN_FILES)
```

Comment:
Good idea to test bad file names! I suggest we use hard-coded strings here. Hard-coded strings will make the cases we are testing much more obvious and more independent (i.e. dropping our dependency on `_csv_to_list`). This great, I'm glad we are thinking seriously about handling the bad input we will inevitably get.

Reply:
Awesome! hardcoding will be readable yes. will do this also.